src/engine/resources.ts

```diff
@@ -279,6 +281,37 @@ export const wandererSources: WandererSource[] = [
     chance: () => [0.5, 0.4, 0.3, 0.2, 0.1, 0.1, 0.1, 0][get("_hipsterAdv")],
     possible: () => myFamiliar() === $familiar`Artistic Goth Kid`,
   },
+  {
+    name: "Comma Goth",
```

Comment:
Did you confirm that this actually works? The wiki says that "as an Artistic Goth Kid, she will not cause special Crayon monster combats."